sunbird-rc-esignet-integration-impl/src/main/java/io/mosip/esignet/sunbirdrc/integration/service/MockVCIssuancePlugin.java

```diff
@@ -0,0 +1,255 @@
+package io.mosip.esignet.sunbirdrc.integration.service;
+
+
+import java.io.StringWriter;
+import java.util.HashMap;
+import java.util.Map;
+import io.mosip.esignet.api.exception.VCIExchangeException;
+import org.apache.velocity.Template;
+import org.apache.velocity.VelocityContext;
+import org.apache.velocity.app.VelocityEngine;
+import org.apache.velocity.runtime.RuntimeConstants;
+import org.apache.velocity.runtime.resource.loader.ClasspathResourceLoader;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+import foundation.identity.jsonld.JsonLDObject;
+import io.mosip.esignet.api.dto.VCRequestDto;
+import io.mosip.esignet.api.dto.VCResult;
+import io.mosip.esignet.api.spi.VCIssuancePlugin;
+import lombok.extern.slf4j.Slf4j;
+
+
+@ConditionalOnProperty(value = "mosip.esignet.integration.vci-plugin", havingValue = "SunbirdRCVCIssuancePlugin")
+@Component
+@Slf4j
+public class MockVCIssuancePlugin implements VCIssuancePlugin {
```

Comment:
for this PR I believe this class should contain interface methods, but throwing as not supported. Also there are lot of commented code, which needs to be removed

Reply:
done